packages/editor-ui/src/modules/ui.ts

```diff
@@ -224,6 +230,9 @@ const module: Module<IUiState, IRootState> = {
 		mappingTelemetry: (state: IUiState) => state.ndv.mappingTelemetry,
 		getCurrentView: (state: IUiState) => state.currentView,
 		isNodeView: (state: IUiState) => [VIEWS.NEW_WORKFLOW.toString(), VIEWS.WORKFLOW.toString(), VIEWS.EXECUTION.toString()].includes(state.currentView),
+		getNDVDataIsEmpty: (state: IUiState) => (panel: 'input' | 'output'): boolean => state.ndv[panel].data.isEmpty,
+		getNDVInputDataIsEmpty: (state: IUiState) => state.ndv.input.data.isEmpty,
+		getNDVOutputDataIsEmpty: (state: IUiState) => state.ndv.output.data.isEmpty,
```

Comment:
please remove `get` from name.. since they don't return functions.. this is a pattern here we need to fix

Reply:
removed store getters and kept the getter function you questioned below